cmd/skupper/skupper.go

```diff
@@ -1482,24 +1490,32 @@ func NewCmdStatusGateway(newClient cobraFunc) *cobra.Command {
 			}
 
 			if len(gateways) == 0 {
-				fmt.Println("No gateway definitions found")
+				l := formatter.NewList()
+				l.Item("No gateway definition found on cluster")
+				gatewayType, err := client.GatewayDetectTypeIfPresent()
+				if err == nil {
+					if gatewayType != "" {
+						l.NewChild(fmt.Sprintf(" A gateway of type %s detected on local host. Run 'skupper gateway delete' to remove.", gatewayType))
```

Comment:
I'm not sure the recommendation to run `gateway delete` is a good idea.  The empty list might be a transient error talking to the gateway pod, and then the user would be removing locally a gateway that is properly defined on the cluster.

Reply:
I agree with your suggestion and will update. The need is for a debug gateway command to inspect and identify cluster/host discrepancies which should be the recommended action here.